api/sys_plugins.go

```diff
@@ -0,0 +1,111 @@
+package api
+
+import (
+	"fmt"
+	"net/http"
+)
+
+// ListPluginsInput is used as input to the ListPlugins function.
+type ListPluginsInput struct{}
+
+// ListPlugins lists all plugins in the catalog and returns their names as a
+// list of strings.
+func (c *Sys) ListPlugins(i *ListPluginsInput) ([]string, error) {
```

Comment:
Please make this a ListPluginsResponse (or ListPluginsOutput) rather than []string.

Reply:
Updated

Reply:
And added tests